run.py

```diff
@@ -470,9 +470,6 @@ def resize():
                 fileObject["height"] = int(match.group(2).strip())
             print "\t (" + str(fileObject["width"]) + " x " + str(fileObject["height"]) + ")"
 
-    if args.end_with != "resize":
-        getKeypoints()
-
```

Comment:
I think the two blank lines are there. At least on the new version on the right >>>